experimental/dds/tree2/src/test/shared-tree/sharedTree.spec.ts

```diff
@@ -1163,7 +1163,7 @@ describe("SharedTree", () => {
 			validateTree(tree2, [expectedState]);
 		});
 
-		it.skip("can rebase cross-field move over unrelated change", () => {
+		it.skip("can rebase cross-field move over delete", () => {
```

Comment:
Remind me, why does this fail again (and are we tracking the work for that anywhere)?

Reply:
I think this was fixed by my previous PR. I'll re-enable the test.